Stripe/STPBankAccount.m

```diff
@@ -24,6 +27,33 @@ @implementation STPBankAccount
 
 @synthesize routingNumber, country, currency, accountHolderName, accountHolderType;
 
+#pragma mark - STPBankAccountStatus
+
++ (NSDictionary<NSString *, NSNumber *> *)stringToStatusMapping {
+    return @{
+             @"new": @(STPBankAccountStatusNew),
+             @"validated": @(STPBankAccountStatusValidated),
+             @"verified": @(STPBankAccountStatusVerified),
+             @"errored": @(STPBankAccountStatusErrored),
+             };
+}
+
++ (STPBankAccountStatus)statusFromString:(NSString *)string {
+    NSNumber *statusNumber = [self stringToStatusMapping][string];
+
+    if (statusNumber) {
+        return (STPBankAccountStatus)[statusNumber integerValue];
+    }
+
+    return STPBankAccountStatusNew;
```

Comment:
We don't actually have an unknown case for `STPBankAccountStatus`. I think the current pattern is to add an "unknown" case only if we think the API is going to add more cases in the near future.